spec/dummy/test/components/stories/kitchen_sink_component_stories.rb

```diff
@@ -10,7 +10,7 @@ class KitchenSinkComponentStories < ViewComponent::Storybook::Stories
       number_pets 2
       sports %w[football baseball]
       select(:favorite_food, { hot_dog: "Hot Dog", pizza: "Pizza", burgers: "Burgers", ice_cream: "Ice Cream" }, "Ice Cream")
-      radio(:mood, { happy: "Happy", sad: "Sad", angry: "Angry", content: "Content" }, "Happy")
+      radio(:mood, ["Happy", "Sad", "Angry", "Content"], "Happy")
```

Comment:
I don't think we need to change the kitchen sink example here. If we have unit tests of the specifics that is enough. Kitchen Sink should stay as the most complicated example of all options. If there is value in detailing this option to take an array lets create a separate story.